tt_eager/tt_dnn/op_library/transformer_tms/multi_core_ssm_eltwise_mul/multi_core_ssm_eltwise_mul.cpp

```diff
@@ -241,7 +251,11 @@ operation::ProgramWithCallbacks multi_core_ssm_eltwise_mul(
                 all_writer_runtime_args[i][2] = num_blocks_written;
             }
 
+            all_writer_runtime_args[i][3] = bshape[2]/TILE_HEIGHT;
+            all_writer_runtime_args[i][4] = hidden_size;
+
             all_compute_runtime_args[i][0] = num_blocks_per_core;
+            all_compute_runtime_args[i][1] = bshape[2]/TILE_HEIGHT;
```

Comment:
Might need to run clang-format on some of these files.

Reply:
Had to do manually